File: src/main/java/com/pauljoda/modularsystems/power/gui/GuiSolidsBank.java
package com.pauljoda.modularsystems.power.gui;

import com.pauljoda.modularsystems.power.container.ContainerSolidsBank;
import com.pauljoda.modularsystems.power.tiles.TileBankSolids;
import net.minecraft.entity.player.InventoryPlayer;

public class GuiSolidsBank extends GuiPowerBase<ContainerSolidsBank> {

   protected TileBankSolids tileEntity;

    public GuiSolidsBank(InventoryPlayer player, TileBankSolids tileEntity) {
        super(new ContainerSolidsBank(player, tileEntity), tileEntity, 175, 165, "inventory.solidspower.title");

        this.tileEntity = tileEntity;
    }

    @Override
    public void addComponents() {}
}

File: src/main/scala/com/teambr/modularsystems/power/gui/GuiSolidsBank.scala
package com.teambr.modularsystems.power.gui

import com.teambr.modularsystems.power.container.ContainerSolidsBank
import com.teambr.modularsystems.power.tiles.TileBankSolids
import net.minecraft.entity.player.EntityPlayer

/**
 * This file was created for Modular-Systems
 *
 * Modular-Systems is licensed under the
 * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License:
 * http://creativecommons.org/licenses/by-nc-sa/4.0/
 *
 * @author Dyonovan
 * @since August 07, 2015
 */
class GuiSolidsBank[C <: ContainerSolidsBank](container : C, player: EntityPlayer, tileEntity: TileBankSolids)
        extends GuiPowerBase[ContainerSolidsBank](container, tileEntity, player.inventory, 175, 165, "inventory.solidspower.title") {

    override def addComponents(): Unit = { }
}
